### ISSUE TITLE
Path.Combine should be used instead

### ISSUE THREAD

**Issue by tomty89** [ImAiiR/QobuzDownloaderX#39]
Currently the code uses the method of string concatenation (i.e. `+` and `\\`) to build paths, which prevents the program from working with (native) mono on Linux. The proper way is to use `Path.Combine()`.

**Comment by tomty89** [comment]
By the way I had problem downloading a full album (when I ran it with wine-mono). Somehow it just timed out and stalled after downloading the second track. Not sure if it was just a temporary network issue or something wrong with the program (or wine).

Fortunately by studying the code I managed to write a bash script (which makes use of `jq` the JSON processor, and `curl` <3) as alternative: https://gist.github.com/tomty89/e1ab603e7404f6b55e7c53331d146f96

Is it true that Berlin is the key? I tried London it failed.

**Comment by ImAiiR** [comment]
Alright, so I've tried to switch things over to `Path.Combine()` now, so if you would like to, test out the [latest version](https://github.com/ImAiiR/QobuzDownloaderX/releases/tag/0.9.9.17), and see if it works for you. This program was made to only run on Windows for me to use myself, so I haven't been testing on any other OS. If it doesn't work and you know the cause, let me know, and I'll see if I can fix it up for the next version.

As for the key you're talking about, I believe some of the values you could use from the .js will result in keys that don't actually work. I think this was done purposefully, but I'm not 100% sure.

**Comment by tomty89** [comment]
It's actually worse now. Not only that it requires an extra dependency (that isn't included). It seems to crash way more often and doesn't even understand the URL, so can't download at all.

I still made a PR that ditches the remaining backslashes used in paths anyway.

**Comment by ImAiiR** [comment]
I've tested the current version and it still understands URLs fine, so I'm not sure what the cause for that issue is. The newest thing I've added was Json.NET, and the .dll for that is included inside of the latest release .zip file, which I've tested. Again I've only tested on windows, so I'm not sure if running through another OS will work.

As for the remaining backslashes, I knew I'd end up forgetting something, so thanks for the PR.
